CONTRIBUTING.md

```diff
@@ -115,3 +115,41 @@
 6. Link the pull request in `issue#<id>`
 
 7. After the pull request is merged, close the `issue#<id>`
+
+
+## Testing
+
+# Integration Testing for Ansible Modules
+
+To perform integration tests for any specific Ansible module such as the `ntnx_vms` module, follow these step-by-step procedures:
+
+## Prerequisites
+- Ensure you are in the collection directory where the module is located.
+
+## Setting up Variables
+1. Navigate to the `tests/integration/targets` directory within the collection.
+2. Fill in the necessary variables inside `prepare_env/vars/main.yml`.
```

Comment:
Please mention that not all vars are compulsory. Also mention different var files for different features like foundation, ndb, etc.